### PR TITLE
[BUG] Fix some details in the fr_FR translation

### PR DESCRIPTION
**Pull request by shtrom**

### CODE REVIEW

File: languages/fr/assess-api.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
     "idleCancelButton": "Oui, je veux continuer !",
     "idleHeadingMsg": "Vous êtes toujours là ?",
     "idleQuitButton": "Quitter",
-    "introItemMessage": "Cliquez sur <b>Démarrer</b> quand vous êtes prêt à commencer l/'activité.",
+    "introItemMessage": "Cliquez sur <b>Démarrer</b> quand vous êtes prêt à commencer l'activité.",
     "item": "Élément",
     "itemCountOf": "sur",
-    "loading": "le chargement",
+    "loading": "Chargement",
```
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "loading": "le chargement",
```

File: languages/fr/assess-api.json [hunk 2]
```diff
@@ -26,37 +26,37 @@
     "retry": "Réessayer",
     "save": "Enregistrer",
     "saveButtonLabel": "Quitter", 
-    "saveError": "Erreur lors de l/'enregistrement.",
+    "saveError": "Erreur lors de l'enregistrement.",
     "saveInProgress": "Enregistrement...",
     "savingInProgress": "Enregistrement en cours.",
-    "saving": "l/'enregistrement",
-    "saved": "enregsitrée",
+    "saving": "enregistrement",
```
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "saving": "l'enregistrement",
```

File: languages/fr/assess-api.json [hunk 3]
```diff
@@ -26,37 +26,37 @@
...
-    "resume": "reprendre",
+    "saveTestSuccessfullyPrompt": "Souhaitez-vous continuer votre {{test}}, ou quitter ?",
+    "resume": "Reprendre",
     "startTest": "Démarrer",
     "submit": "Envoyer",
     "submitButtonLabel": "Terminer",
     "submitComplete": "Envoyé",
     "submittingInProgress": "Envoi en cours...",
-    "submitting": "l/'envoi",
+    "submitting": "envoi",
```
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "submitting": "l'envoi",
```

File: languages/fr/assess-api.json [hunk 4]
```diff
@@ -86,16 +86,16 @@
...
-    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche plus (+).",
-    "zoomOutResult": "Le navigateur diminue le zoom lorsque vous appuyez sur la touche Soustraction (-).",
+    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche Plus (+).",
+    "zoomOutResult": "Le navigateur diminue le zoom lorsque vous appuyez sur la touche Moins (-).",
     "collapseMenu": "Réduire le menu",
-    "submitActionConfirmation": "Êtes-vous sûr de vouloir {{action}} l/'{{test}}",
-    "submitWithUnattemptedQuestion":"Vous n/'avez pas répondu à {{unattemptedResponsesCount}} question.",
-    "submitWithUnattemptedQuestions":"Vous n/'avez pas répondu à {{unattemptedResponsesCount}} questions.",
-    "isAllResponsesAttempted":"{{allQuestionsAttempted}}. Voulez-vous {{action}} l/'{{test}} maintenant ?",
+    "submitActionConfirmation": "Êtes-vous sûr de vouloir {{action}} le {{test}}",
```
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "submitActionConfirmation": "Êtes-vous sûr de vouloir {{action}} votre {{test}}",
```

File: languages/fr/assess-api.json [hunk 5]
```diff
@@ -86,16 +86,16 @@
...
+    "submitWithUnattemptedQuestion":"Vous n'avez pas répondu à 1 question.",
+    "submitWithUnattemptedQuestions":"Vous n'avez pas répondu à {{unattemptedResponsesCount}} questions.",
+    "isAllResponsesAttempted":"{{allQuestionsAttempted}}. Voulez-vous {{action}} le {{test}} maintenant ?",
```
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "isAllResponsesAttempted":"{{allQuestionsAttempted}}. Voulez-vous {{action}} votre {{test}} maintenant ?",
```

**Comment by shtrom** [comment]
We'll have to squash this all before merging.

File: languages/fr/assess-api.json [hunk 6]
```diff
@@ -26,37 +26,37 @@
...
     "saveInProgress": "Enregistrement...",
     "savingInProgress": "Enregistrement en cours.",
-    "saving": "l/'enregistrement",
-    "saved": "enregsitrée",
-    "saveTestSuccessfully": "Votre activité a bien été enregistrée",
-    "saveTestSuccessfullyPrompt": "Souhaitez-vous continuer l/'{{test}}, ou quitter ?",
-    "resume": "reprendre",
+    "saving": "l'enregistrement",
+    "saved": "enregistrée",
+    "saveTestSuccessfully": "Votre {{test}} a bien été enregistrée",
```
Comment (SalimBensiali) [inline languages/fr/assess-api.json]: @shtrom, if we use {{test}} here, don't we need to use `enregistré(e)`?

Comment (shtrom) [reply languages/fr/assess-api.json]: I was just ranting at Edwin about it, but missed that case. Cheers!

Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "saveTestSuccessfully": "Votre activité a bien été enregistrée",
```

File: languages/fr/assess-api.json [hunk 7]
```diff
@@ -26,37 +26,37 @@
...
+    "saveError": "Erreur lors de l'enregistrement.",
     "saveInProgress": "Enregistrement...",
     "savingInProgress": "Enregistrement en cours.",
-    "saving": "l/'enregistrement",
-    "saved": "enregsitrée",
-    "saveTestSuccessfully": "Votre activité a bien été enregistrée",
-    "saveTestSuccessfullyPrompt": "Souhaitez-vous continuer l/'{{test}}, ou quitter ?",
-    "resume": "reprendre",
+    "saving": "l'enregistrement",
+    "saved": "enregistrée",
```
Comment (SalimBensiali) [inline languages/fr/assess-api.json:33]: This one, not knowing the context, could be both `enregistré` or `enregistrée`

File: languages/fr/assess-api.json [hunk 8]
```diff
@@ -26,37 +26,37 @@
...
     "tryAgain": "Réessayer",
-    "idleSaveError": "Une erreur est survenue pendant l/'enregistrement.",
-    "errorOccurredWhile": "Une erreur est survenue pendant",
-    "submitFailed": "Votre activité n/'a pas pu être enregistrée. Vérifiez votre connexion réseau et réessayez.",
-    "submitWarningMsg": "Votre activité a mise en pause par un administrateur, et quittera automatiquement dans ",
-    "submitFailedWithOnlyAccessRawDataOption":"Si le problème persiste, séléctionnez et copiez le texte ci-dessous, et envoyez le à votre administrateur. Ce texte contient les données de votre activité et vos réponses.",
-    "submitFailedWithOtherOptions": "Si le problème persiste, choisissez une option pour accéder aux données complètes de votre activité et vos réponses.",
+    "idleSaveError": "Une erreur est survenue pendant l'enregistrement.",
+    "errorOccurredWhile": "Une erreur est survenue pendant ",
+    "submitFailed": "Votre activité n'a pas pu être soumise. Vérifiez votre connexion réseau et réessayez.",
```
Comment (SalimBensiali) [inline languages/fr/assess-api.json]: How about "Veuillez vérifier"? That would sound more polite to me.

File: languages/fr/assess-api.json (hunk 7 above)
Comment (shtrom) [reply languages/fr/assess-api.json:33]: This is an annoying one... I don't know when this label is used, and looking at the rest of the file, I'm not sure it's used. Nontheless, Angélique had it like that, I was only fixing a typo, and it's likely that the `{{test==activité}} a été {{saved==enregistrée}}`. So I agree it's potentially incorrect, but I think it's safe at the moment.

File: languages/fr/assess-api.json (hunk 8 above)
Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "submitFailed": "Votre activité n'a pas pu être soumise. Veulliez vérifier votre connexion réseau et réessayez.",
```

File: languages/fr/assess-api.json [hunk 9]
```diff
@@ -68,15 +68,15 @@
...
+    "fontSizeInstructions": "Ajuster la taille de police dans l'activité",
     "small": "Petit",
     "normal": "Normal",
     "large": "Grand",
     "xlarge": "Très grand",
     "xxlarge": "Énorme",
     "zoomInstructions": "Vous pouvez effectuer un zoom avant ou arrière en utilisant les raccourcis clavier suivants:",
-    "command": "Cmd",
-    "control": "Ctrl",
+    "command": "Command",
```
Comment (SalimBensiali) [inline languages/fr/assess-api.json]: I was gonna say how about "Pomme" which I heard and read more than once, but then saw this
image

It's "Command**e**" as in "La commande"

Comment (shtrom) [inline languages/fr/assess-api.json]:
```suggestion
    "command": "Commande",
```

File: languages/fr/assess-api.json [hunk 10]
```diff
@@ -86,16 +86,16 @@
     "plus": "+",
     "minus": "-",
     "zero": "0",
-    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche plus (+).",
-    "zoomOutResult": "Le navigateur diminue le zoom lorsque vous appuyez sur la touche Soustraction (-).",
+    "zoomInResult": "Le navigateur effectuera un zoom avant lorsque vous appuyez sur la touche Plus (+).",
```
Comment (SalimBensiali) [inline languages/fr/assess-api.json:89]: How about "zoomer" / "dé-zoomer"? Looking at some references such as https://www.commentcamarche.net/faq/42939-comment-faire-un-zoom-sur-mac

Comment (shtrom) [reply languages/fr/assess-api.json:89]: Don't know about this one. I find “dé-zoomer” a bit ugly, and “zoom avant/arrière” feels familiar.

Comment (angelique-dubois-lrn) [reply languages/fr/assess-api.json:89]: I used linguee.com a lot to try and find the most popular translations of some words, and zoom avant/arrière seemed to appear more often. Zoom doesn't even appear in the Académie Française's dictionary, so I assume zoom avant/arrière might be preferred in a school context.

**Review by angelique-dubois-lrn — APPROVED** [review]

**Pull request merged**